common/lib/xmodule/xmodule/course_module.py

```diff
@@ -390,44 +390,46 @@ class CourseFields(object):
         scope=Scope.settings,
         default=[
             {
-                "short_description": _("Getting Started With Studio"),
+                "short_description": _("Course Introduction"),
                 "items": [
                     {
-                        "short_description": _("Add Course Team Members"),
+                        "short_description": _("Welcome Message"),
                         "long_description": _(
-                            "Grant your collaborators permission to edit your course so you can work together."
+                            "Provide a welcome message on course info page at the beginning of the course."
                         ),
                         "is_checked": False,
-                        "action_url": "ManageUsers",
-                        "action_text": _("Edit Course Team"),
+                        "action_url": "CourseUpdates",
+                        "action_text": _("View Course Updates"),
                         "action_external": False,
                     },
                     {
-                        "short_description": _("Set Important Dates for Your Course"),
+                        "short_description": _("Direct learners to edX DemoX"),
                         "long_description": _(
-                            "Establish your course's student enrollment and launch dates on the Schedule and Details "
-                            "page."
+                            "Provide a link to edX DemoX course (an introduction to the edX platform for new learners) in the "
```

Comment:
Include a link to edX DemoX, the online course that introduces learners to the edX system, in the first week of the course.